dime-coding-standards/checklists/tex/Reviewer Feedback Checklist

```diff
@@ -0,0 +1,154 @@
+        
+		\section*{Overview - First Impression}
+
+        Once you have confirmed that you received a package from your code review partner, please refer to the following list for basic recommended practices:
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}		
+			
+			\item[] \dimeCheckBox{ Scripts for data cleaning, variable creation, and analysis are separate; and analysis scripts do not include any data processing, unless necessary for the creation of a table or graphic}
+			\item[] \dimeCheckBox{ Variable construction scripts include detailed comments about each variable created}
+			\item[] \dimeCheckBox{ Analysis scripts do not depend on having the results of other scripts in memory, except for the Master script}
+			\item[] \dimeCheckBox{ All scripts are well-commented and formatted, such that one can easily identify functional chunks of code and evaluate whether they correctly implement the econometric or statistical process described}
+			\item[] \dimeCheckBox{ Tables are output as .csv or .tex files or other raw text files when possible}
+		\end{itemize}
+	
+		\vspace{.6em}
+
+        \section*{Reproducibility}
+
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{Code ran perfectly on the first run \textbf{(Applicable only if data was shared with you.)}}
+			\item[] \dimeCheckBox{Outputs shared (if any) replicated perfectly on the first run}
+            \item[] \dimeCheckBox{If not, was able to determine the reason for why outputs changed}
+			\item[] \dimeCheckBox{Make note of any issues you faced in these steps. You will need to provide this information in \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed review summary form.}}
+			
+		\end{itemize}
+		
+        \vspace{.6em}
+
+       \section*{Ease of Use and Understanding}
+
+       
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{Understood the code from original documentation and README file}
+			\item[] \dimeCheckBox{Package needed additional information}
+            \item[] \dimeCheckBox{Understood the purpose of the code}
+			\item[] \dimeCheckBox{As a new RA joining this project right now, will need additional support to be able to contribute to this code}
+                \item[] \dimeCheckBox{Make note of what additional materials would help. You will need to provide this information in \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed review summary form.}}
+                \item \dimeTextField{time}{1 cm}{\textbf{Approximate time} needed to understand the code well enough to contribute \textbf{(0-3 days)}: }
+                \item \dimeTextField{time}{2.5 cm}{How likely is it that you would need to make multiple changes all over the code if you were to change a core task \textbf{\textit{(Don't know, very unlikely, unlikely, somewhat likely, very likely)}}:}
```

Comment:
reword --> 

- [ ] the code is organized such that if I had to change a core task I could do it in only one place